hts.c

```diff
@@ -2575,6 +2574,8 @@ hts_itr_multi_t *hts_itr_regions(const hts_idx_t *idx, hts_reglist_t *reglist, i
             }
 
             itr->reg_list[i].tid = getid(hdr, reglist[i].reg);
+            if (itr->reg_list[i].tid < 0) 
+                fprintf(stderr,"[hts_itr_regions]: region '%s' specifies an unknown reference name. Continue anyway.\n", reglist[i].reg);
```

Comment:
This should be using ```hts_log_error``` or ```hts_log_warning``` instead of fprintf, which then gives control over whether it is displayed.